DeepSpeech.py

```diff
@@ -1121,6 +1129,7 @@ def start_coordination(self, data_sets, step=0):
             # We only have to train, if we are told so and are not at the target epoch yet
             self._train = FLAGS.train and self._target_epoch > self._epoch
             self._test = FLAGS.test
+            self._dev = FLAGS.dev
```

Comment:
From the old code it is already clear, when a validation runs - you don't have to deal with that at all. Things should be fine. The only thing you have to care about should be the stopping.